### PR TITLE
feat(members.js): add a 'Resend Invite' button

### PR DESCRIPTION
**Pull request by alurban**
Fixes opencollective/opencollective#5030
Require opencollective/opencollective-api#7429

## Description

Introduce a 'Resend Invite' button for invited team members in a given collective. This button allows users to resend invitations to prospective members on command, and includes the following changes:

* Adapt the `inviteMemberMutation` and graphQL query for class `Members`
* Include a helper function, `resendInvite`, to wrap the call to this mutation as an anonymous `onClick` action
* Include E2E tests for the new button
* Run `build:langs`

## Screenshots

Screen Shot 2022-04-20 at 4 32 26 PM

### CODE REVIEW

File: test/cypress/integration/22-collective.edit.test.js [hunk 1]
```diff
@@ -56,6 +57,12 @@ describe('edit collective', () => {
     cy.getByDataCy('create-collective-mini-form').should('not.exist'); // Wait for form to be submitted
     cy.getByDataCy('confirmation-modal-continue').click();
     cy.get('[data-cy="member-1"] [data-cy="member-pending-tag"]').should('exist');
+    cy.getInbox().should('have.length', 2);
+
+    // Re-send the invitation email
+    cy.getByDataCy('resend-invite-btn').should('exist').first().click({ force: true });
+    cy.wait(200); // Wait for email
+    cy.getInbox().should('have.length', 3);
```
Comment (Betree) [inline test/cypress/integration/22-collective.edit.test.js:65]: Nice :+1:

**Review by Betree — COMMENTED** [review]
I haven't tested it yet but the code looks good to me

File: test/cypress/integration/22-collective.edit.test.js (hunk 1 above)
Comment (alurban) [reply test/cypress/integration/22-collective.edit.test.js:65]: I love how straightforward it was to design this test

**Review by Betree — COMMENTED** [review]
When users click on "Resend Invite", there is no visual feedback to indicate that the invitation was sent. Since we don't show any loading state on the button either it feels like it has no effect.

Peek 2022-05-24 14-50

**Comment by Betree** [comment]
To track in a followup issue: users shouldn't be allowed to spam the resend button. Ideally, there should be a timer that lets you click it again after something like 60 seconds.

Peek 2022-05-24 15-07

**Review by Betree — APPROVED** [review]
Resolved the abovementioned feedback by using the same approach as `components/expenses/ExpenseInviteNotificationBanner.js`

**Pull request merged**